infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/receive/LogReplicationMetadataManager.java

```diff
@@ -298,13 +298,13 @@ public void setSnapshotApplied(LogReplicationEntry entry) {
     }
 
     public String getMetadata() {
```

Comment:
This is intended to be a friendly print of the metadata, right? can we move it to the override of the toString?